efficientdet/loss.py

```diff
@@ -25,8 +25,11 @@ def calc_iou(a, b):
 
 
 class FocalLoss(nn.Module):
-    def __init__(self):
+    def __init__(self, matched_threshold, unmatched_threshold, negatives_lower_than_unmatched=False):
```

Comment:
maybe set `matched_threshold, unmatched_threshold` with default parameters, so that this loss function can be used somewhere else out-of-the-box

Reply:
Good idea - done.